### PR TITLE
Update shiftstack-team 2021-10-06

### PR DESCRIPTION
**Pull request by mdbooth**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -3,7 +3,7 @@
 aliases:
   shiftstack-team:
     - EmilienM
-    - Fedosin
+    - stephenfin
```
[the remark this answers is not in the document]
Comment (mdbooth) [reply OWNERS_ALIASES]: Sorted

**Review by mandre — COMMENTED** [review]
/lgtm

**Pull request merged**